---
layout: page
title: R tutorials
permalink: /r-tutorials/
---
The AmphiBac database relies on open-source tools (R, GitHub, and vsearch) to provide access to and useage of the database. While open-source software is the bee's knees, there can often time be a learning curve as to how to best use these tools for users of all levels. As such, we have provided step-by-step tools to guide you through the use of the R-package with as much explanation as we can provide. 

---
<ol>

1. From raw reads to database ready analysis (for beginners of high-throughput sequencing analysis of 16S rRNA gene data)
2. Importing your data into R from common bioinformatics pacakges (e.g. QIIME2, mothur, etc)
3. Comparing your sequences to the AmphiBac Database
4. Calculate the percentage of inhibitory/enhancing/noninhibitory taxa 
5. Create a ASV/OTU table of inhibitory taxa
6. Antifungal richness calculations
7. Implement filtering to subset database based on metadata (e.g. country, frog species)
8. Pull the latest AmphiBac database (one is provided on CRAN, but we do update it inbetween CRAN releases...we're good like that)  
</ol>
